src/it/tita/src/test/resources/json-module-app-pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-json-module</artifactId>
-            <version>2.4.0-SNAPSHOT</version>
+            <version>2.5.0</version>
```

Comment:
Shouldn't it be `${json.module.version}` ?

Reply:
We can definitely switch to a property, but keep in mind that we still have to specify it both in this file as well as in JSON Module.